src/autoscaler/api/config/config.go

```diff
@@ -267,6 +276,24 @@ func LoadConfig(filepath string, vcapReader configutil.VCAPConfigurationReader)
 	return &conf, nil
 }
 
+func FromJSON(data []byte) (*Config, error) {
```

Comment:
Do we need these helper functions in other components ? They seems to like helper functions and therefore, should be somewhere else e.g helper suite

Reply:
you are right about this, but we should do it once we use it in 2 places. So once we move the next component we can do something with generics, because each config model would be different.